scripts/run-pallet-benchmarks.sh

```diff
@@ -17,4 +17,5 @@ cargo run \
     --extrinsic '*' \
     --repeat 20 \
     --steps 50 \
-    --output ./pallets/$pallet/src/raw-weights.rs
+    --output ./pallets/$pallet/src/raw-weights.rs \
+    --template ./.maintain/frame-weight-template.hbs
```

Comment:
what is this for?

Reply:
Matches the command used in github actions and makes the diff on the weights file way easier to read.
https://github.com/OAK-Foundation/OAK-blockchain/blob/19dc34639575ef8371b27520fcb9c96dc403df3b/.github/workflows/benchmark.yml#L61